quay/quay-operator
# PROJQUAY-828 - Downstream Component Images

**Issue:** https://issues.redhat.com/browse/PROJQUAY-828
https://issues.redhat.com/browse/PROJQUAY-828

**Changelog:** Add initial downstream release Kustomize overlay for managed component container images.

**Docs:** Ensure that users know how to create `redhat-pull-secret` ([documentation exists](https://access.redhat.com/solutions/3533201)).

**Testing:** 
1. Create `redhat-pull-secret` in same namespace as `QuayRegistry`.
2. Create `QuayRegistry` with some managed components.

Expected:
- All containers pull successfully.

**Details:** Adds Kustomize overlays for an initial downstream release using official Red Hat container images for relevant managed components. These container images require a [Red Hat pull secret](https://access.redhat.com/solutions/3533201).
